com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/clients/ServiceGetter.java

```diff
@@ -2,35 +2,39 @@
 
 import java.util.Map;
 
+import javax.inject.Inject;
+
 import org.cloudfoundry.client.lib.CloudControllerClient;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
 @Component
 public class ServiceGetter {
 
-    @Autowired
-    @Qualifier("serviceInstanceGetter")
     private AbstractServiceGetter serviceInstanceGetter;
-    
-    @Autowired
-    @Qualifier("userProvidedServiceInstanceGetter")
     private AbstractServiceGetter userProvidedServiceInstanceGetter;
-    
+
+    @Inject
+    public ServiceGetter(@Qualifier("serviceInstanceGetter") AbstractServiceGetter serviceInstanceGetter,
+        @Qualifier("userProvidedServiceInstanceGetter") AbstractServiceGetter userProvidedServiceInstanceGetter) {
```

Comment:
It is hard to read inline @Qualifier annotations. So, here you could inject the exact type??